src/cli.js

```diff
@@ -118,8 +118,13 @@ export default function cli(args, log, error, exit) {
       skipMissing: opt.argv.skipMissing,
     }))
     .then(result => print(result, log, opt.argv.json))
-    .then(({ dependencies: deps, devDependencies: devDeps }) =>
-      exit(opt.argv.json || (deps.length === 0 && devDeps.length) === 0 ? 0 : -1))
+    .then(({ dependencies: deps, devDependencies: devDeps, missing }) => {
+      const hasNoErrors = deps.length === 0 &&
+                          devDeps.length === 0 &&
+                          Object.keys(missing).length === 0;
```

Comment:
Could you please wrap this check with `()` and each check on each line?

```
const hasError = (
  deps.length !== 0 ||
  devDeps.length !== 0 ||
  Object.keys(missing).length !== 0
);
```

Reply:
I just updated that, I used `> 0` instead of `!== 0` because it feels more idiomatic to me, if that's fine with you :)